src/utils/web3_tools.js

```diff
@@ -116,3 +117,11 @@ export function link(contract, name, library) {
   contract.bytecode = contract.bytecode.replace(pattern, address);
 }
 
+export async function getBalance(web3, address) {
+  return new BN(await web3.eth.getBalance(address));
```

Comment:
Functions should return POJSO as a rule (In this case a string). For context: we already had a lot of trouble when some functions returned a BN and some others have returned a string.